package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.2",
```

Comment:
Why?

Reply:
Because this component present in your project requires it https://github.com/status-im/status-components/blob/master/src/components/Welcome.jsx#L2

Reply:
It's a copy/paste mistake. Do you mind removing the dependency and adapt the component?

Reply:
What do you mean with "adapt the component"?

Reply:
I mean remove all the material stuff. This is not a real component, just placeholder.